panther_driver/src/driver_node.py

```diff
@@ -200,10 +210,10 @@ def driverNode():
     robot_y_pos = 0.0
     robot_th_pos = 0.0
 
-    wheel_FL_ang_pos_last = 0.0
-    wheel_FR_ang_pos_last = 0.0
-    wheel_RL_ang_pos_last = 0.0
-    wheel_RR_ang_pos_last = 0.0
+    wheel_FL_ang_pos = 0.0
+    wheel_FR_ang_pos = 0.0
+    wheel_RL_ang_pos = 0.0
+    wheel_RR_ang_pos = 0.0
```

Comment:
Generally, in my opinion, a more convenient way of playing with variables like these would be packing them into a container like a dictionary in python. 
You would only create here wheels_angular_position = {"FL": 0, "FR": 0, ...}
Pros for this:
- when you calculate each position you can use for loop
- when assigning to joint_state_msg.position you can simply type
`joint_state_msg.position = list(wheels_angular_position.values())`
instead of
`joint_state_msg.position = [
                wheel_FL_ang_pos,
                wheel_FR_ang_pos,
                wheel_RL_ang_pos,
                wheel_RR_ang_pos
            ]`

Reply:
By definition dictionaries are not ordered, so in theory `wheels_angular_position.values()` might not result in correct order of the output.

Anyway, I made changes to use lists for the computations.

Reply:
Starting from Python 3.7 dictionaries are ordered. If we need to use a lower version because of the ROS1, there is an optional OrderedDict from the "collections" module.

Reply:
Let's just leave it as it already is. Your suggestion might require too many changes in the files.